homeassistant/helpers/llm.py

```diff
@@ -416,9 +414,7 @@ def _async_get_tools(
                 ):
                     continue
 
-                script_tool = ScriptTool(self.hass, state.entity_id)
-                if script_tool.parameters.schema:
```

Comment:
Do you remember why exposed scripts without a schema wouldn't get exposed as a tool? Seems like it can have unintended side effects

Reply:
It is to optimize the usage of the context. LLMs have no problems running the scripts with `HassTurnOn`, and a script description as a tool takes more context than just listing it.
We did it in https://github.com/home-assistant/core/pull/123061